src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig

```diff
@@ -3,85 +3,87 @@
 {% block page_title 'Security' %}
 
 {% block toolbar %}
-    {% if collector.token %}
-        {% set is_authenticated = collector.enabled and collector.authenticated  %}
-        {% set color_code = is_authenticated ? '' : 'yellow' %}
-    {% elseif collector.enabled %}
-        {% set color_code = collector.authenticatorManagerEnabled ? 'yellow' : 'red' %}
-    {% else %}
-        {% set color_code = '' %}
-    {% endif %}
-
-    {% set icon %}
-        {{ include('@Security/Collector/icon.svg') }}
-        <span class="sf-toolbar-value">{{ collector.user|default('n/a') }}</span>
-    {% endset %}
-
-    {% set text %}
-        {% if collector.impersonated %}
-            <div class="sf-toolbar-info-group">
-                <div class="sf-toolbar-info-piece">
-                    <b>Impersonator</b>
-                    <span>{{ collector.impersonatorUser }}</span>
-                </div>
-            </div>
+    {% if not collector.firewall %}
```

Comment:
Why `if not`? I would have expected `if collector.firewall` instead.

Reply:
Whoa, that's indeed a big typo I made when fixing the complex conflicts.

Reply:
Fixed in e1b352ebe0